src/main/java/Mikey.java

```diff
@@ -0,0 +1,295 @@
+    }
+
+    public static Task newEvent(String taskName, String timeOfEvent, int completion) {
+        Task newTask = null;
+        newTask = new Event(taskName, timeOfEvent);
+        if(completion == 1) {
+            newTask.isDone = true;
+        } else {
+            newTask.isDone = false;
+        }
+        tasks.add(newTask);
+        return newTask;
+    }
+    public static void printTask(int taskNumber) {
+        System.out.println("[" + tasks.get(taskNumber).getTaskType() + "]" + "["
+                + tasks.get(taskNumber).getStatusIcon() + "] " +  tasks.get(taskNumber).getName() + " "
+                    + tasks.get(taskNumber).getDate());
+        System.out.println();
+    }
+
+    public static void addTaskMessage() {
+        System.out.println("Now yous got " + tasks.size() + " thing(s) ta do in ya list");
+        System.out.println("Anythin else ya need me ta help ya with?");
+        System.out.println();
+    }
+
+    public static void deleteTask(int taskNumber) {
+        System.out.println("I've gotcha mate, removed [" + tasks.get(taskNumber).getTaskType() + "]" + "["
+            + tasks.get(taskNumber).getStatusIcon() + "]" + tasks.get(taskNumber).getName() + " "
+                + tasks.get(taskNumber).getDate());
+        tasks.remove(taskNumber);
+    }
+
+    public static void saveToFile() throws java.io.IOException {
+        StringJoiner taskFormatter = new StringJoiner(System.lineSeparator());
+        for (Task t: tasks) {
+            taskFormatter.add(t.taskStringFormat());
+        }
+        int count = 0;
+        int maxTries = 1;
+        while (true) {
+            try {
+                FileWriter listWrite = new FileWriter("./data/Mikey.txt");
+                listWrite.write(taskFormatter.toString());
+                listWrite.close();
+                break;
+            } catch (FileNotFoundException e) {
+                fileNotFound(taskFormatter.toString());
+            }
+        }
+    }
+
+    public static void readFromFile() throws java.io.IOException{
+        try {
+            File readingFile = new File("./data/Mikey.txt");
+            Scanner fileScan = new Scanner(readingFile);
+            while(fileScan.hasNextLine()) {
+                String taskLine = fileScan.nextLine();
+                String[] taskDetails = taskLine.split(" | ", 0);
+                String taskType = taskDetails[0];
+                String taskCompletion = taskDetails[2];
+                String taskDescription = taskDetails[4];
+                if(taskType.equalsIgnoreCase("T")) {
+                    try {
+                        newTodo(taskDescription, Integer.parseInt(taskCompletion));
+                    } catch (StringIndexOutOfBoundsException e) {
+                        System.out.println("Oi mate, I can't create an empty task yea?");
+                        System.out.println();
+                    }
+                } else if (taskType.equalsIgnoreCase("D")) {
```

Comment:
Avoid deep nesting, it will improve the readability of the code. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-deep-nesting)